protocol/x/clob/types/constants.go

```diff
@@ -8,6 +8,10 @@ import (
 // `MsgPlaceOrder` or `MsgCancelOrder` message will be considered valid by the validator.
 const ShortBlockWindow uint32 = 20
 
+// MaxMsgBatchCancelBatchSize represents the maximum number of cancels that a MsgBatchCancel
+// can have in one Msg.
+const MaxMsgBatchCancelBatchSize uint32 = 100
```

Comment:
for my understanding - did we decide to set to to 100 or 1000?

Reply:
https://dydx-team.slack.com/archives/C06JZMTFACV/p1708985170576359?thread_ts=1708970027.055389&cid=C06JZMTFACV

We can change this as needed later.